app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, file_read, file_write = command.split()
```

Comment:
Before check that command have 3 params. Because if you do: copy_file("cp") it will raise an error